public/user/css/main.css

```diff
@@ -12,6 +12,11 @@ body {
   position: relative;
   font-weight:400px;
 }
+@media (max-width: 480px) {
```

Comment:
I think bootstrap provides a way to do this. http://getbootstrap.com/css/#responsive-utilities-classes
Angular can also help you with this: https://material.angularjs.org/latest/layout/introduction

But you can use media queries is you want.